libs/blocks/quiz-results/quiz-results.js

```diff
@@ -0,0 +1,95 @@
+import { createTag } from '../../utils/utils.js';
+import { getMetadata, handleStyle } from '../section-metadata/section-metadata.js';
+
+async function loadFragments(el, experiences) {
+  // eslint-disable-next-line no-restricted-syntax
+  for (const href of experiences) {
+    /* eslint-disable no-await-in-loop */
+    const a = createTag('a', { href });
+    el.append(a);
+    const { default: createFragment } = await import('../fragment/fragment.js');
+    await createFragment(a);
+  }
+}
+
+function redirectPage(quizUrl, debug, message) {
+  const url = (quizUrl) ? quizUrl.text : 'https://adobe.com';
+  window.lana.log(message);
+
+  if (debug === 'quiz-results') {
+    // eslint-disable-next-line no-console
+    console.log(`${message}, redirecting to: ${url}`);
+  } else {
+    window.location = url;
+  }
+}
+
+function setAnalytics(hashValue, debug) {
+  /* eslint-disable no-underscore-dangle */
+  window.alloy_load ??= {};
+  window.alloy_load.data ??= {};
+  window.alloy_all ??= {};
+  window.alloy_all.data ??= {};
+  window.alloy_all.data._adobe_corpnew ??= {};
+  window.alloy_all.data._adobe_corpnew.digitalData ??= {};
+  window.alloy_all.data._adobe_corpnew.digitalData.page ??= {};
+  window.alloy_all.data._adobe_corpnew.digitalData.page.pageInfo ??= {};
+  window.alloy_all.data._adobe_corpnew.digitalData.page.pageInfo.customHash = hashValue;
+  if (debug === 'quiz-results') {
+    // eslint-disable-next-line no-console
+    console.log(`Setting a custom hash for pageload to: ${window.alloy_all.data._adobe_corpnew.digitalData.page.pageInfo.customHash}`);
+  }
+}
+
+export const loadingErrorText = 'Could not load quiz results:';
```

Comment:
super nit (can leave as is if you want) - string constants are usually all caps and declared at top of file
```suggestion
export const LOADING_ERROR = 'Could not load quiz results:';
```

Reply:
This was put in place in service of unit tests to reduce repeated strings there. In that context, I opted not to go with all caps and I'd need to update the references to it in the test as well. If it's not a big deal, I'd prefer to leave it as is.